languages/python/oso/oso/oso.py

```diff
@@ -38,3 +38,24 @@ def is_allowed(self, actor, action, resource) -> bool:
             return True
         except StopIteration:
             return False
+
+    def get_allowed_actions(self, actor, resource, allow_wildcard=False):
```

Comment:
Needs a proper docstring. Especially to describe the wildcard behaviour.

Reply:
Done.